roachpb/metadata.pb.go

```diff
@@ -92,9 +92,9 @@ func (*RangeTreeNode) ProtoMessage()    {}
 
 // StoreCapacity contains capacity information for a storage device.
 type StoreCapacity struct {
-	Capacity   int64 `protobuf:"varint,1,opt,name=Capacity" json:"Capacity"`
-	Available  int64 `protobuf:"varint,2,opt,name=Available" json:"Available"`
-	RangeCount int32 `protobuf:"varint,3,opt,name=RangeCount" json:"RangeCount"`
+	Capacity   int64 `protobuf:"varint,1,opt,name=capacity" json:"capacity"`
```

Comment:
Yes it will. The associated typescript type is in `ui/ts/models/proto.ts`, on the line `export interface StoreCapacity`.  Just need to lower-case all of those members, then they typescript compiler will show you everywhere else you need to change.